multiplier_analysis/nbconverted/1_get_multiplier_LV_coverage.py

```diff
@@ -7,7 +7,7 @@
 # 
 # The PLIER model performs a matrix factorization of gene expression data to get two matrices: loadings (Z) and latent matrix (B). The loadings (Z) are constrained to aligned with curated pathways and gene sets specified by prior knowledge [Figure 1B of Taroni et. al.](https://www.cell.com/cell-systems/pdfExtended/S2405-4712(19)30119-X). This ensure that some but not all latent variables capture known biology. The way PLIER does this is by applying a penalty such that the individual latent variables represent a few gene sets in order to make the latent variables more interpretable. Ideally there would be one latent variable associated with one gene set unambiguously.
 # 
-# While the PLIER model was trained on specific datasets, MultiPLIER extended this approach to all of recount2, where the latent variables should correspond to specific pathways or gene sets of interest. Therefore, we will look at the coverage of generic genes versus specific genes across these MultiPLIER latent variables.
+# While the PLIER model was trained on specific datasets, MultiPLIER extended this approach to all of recount2, where the latent variables should correspond to specific pathways or gene sets of interest. Therefore, we will look at the coverage of generic genes versus other genes across these MultiPLIER latent variables, which represent biological patterns.
```

Comment:
So just to be clear, what is the difference between "specific genes" and "other genes"? Are specific genes the ones that were found to be differentially expressed in one or a few experiments in your analyses, and "other" genes are just all the genes that weren't found to be generic (i.e. not necessarily DE in any experiment)? Might be good to add a brief description of this difference here in the header.

Reply:
Yes that is correct. Other (or non-generic) genes is a superset that will include specific genes (i.e. genes that are DE in certain experiments) and housekeeping genes (i.e. genes that are consistently unchanged across experiments). I will definitely add text to clarify this. I can see my future self being confused.